lib/iblinter/plugin.rb

```diff
@@ -61,6 +62,11 @@ def iblinter_installed?
         return true
       end
 
+      unless @execute_command.nil?
+        system(@execute_command)
```

Comment:
We have to execute to check commands are valid. It may cause side effects. How do you think?

Reply:
Ummm, I agree that we have to check given command are executable but IMO, we can't ensure that for completely unknown command before execute it.
If you want to support completely execute_command, user must give block to check if the command is valid like below.
```ruby
iblinter.execute_command = ExecuteCommand.new(
  "mint run iblinter",
  installed: proc do
    `mint list | grep iblinter`
  end
)

```

But If you think this is too much, just skipping to check when executable_command is given is OK.